src/application/services/useNote.ts

```diff
@@ -96,7 +96,7 @@ export default function (options: UseNoteComposableOptions): UseNoteComposableSt
     /**
      * @todo try-catch domain errors
      */
-    note.value = await noteService.getNoteById(id);
+    note.value = (await noteService.getNoteById(id)).note;
```

Comment:
at this moment, access right are ignored. Let's set them to a separate variable next to `note` so it could be available in a component. Then, use this variable to pass `readOnly: true` to the Editor.js if `canEdit=false`